person.rb

```diff
@@ -2,14 +2,15 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1...1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- Kindly make sure a method for adding rentals is present. 
Hint: ```
`def add_rental(person, date)`
```
- Please make sure the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
Hint: ```
`Rental.new(date, self, person)`
```

Reply:
## Thank you @rotimiazeez